README.md

```diff
@@ -4,48 +4,34 @@
  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/PegasysEng/teku/blob/master/LICENSE)
  [![Gitter chat](https://badges.gitter.im/PegaSysEng/teku.png)](https://gitter.im/PegaSysEng/teku)
 
-Implementation of the Ethereum 2.0 Beacon Chain.
+Teku is a Java implementation of the Ethereum 2.0 Beacon Chain.
 
-Based on the (evolving) [specification](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md).
+## Useful links 
 
-## Build Instructions
-
-### Install Prerequisites
-
-**1) Java 11**
+* [Ethereum 2.0 Beacon Chain specification](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md) 
+* [Teku user documentation](https://docs.teku.pegasys.tech/en/latest/)
+* [Teku REST API reference documentation](https://pegasyseng.github.io/teku/latest/)
+* [Teku issues](https://github.com/PegaSysEng/teku/issues)
+* [Contribution guidelines](CONTRIBUTING.md)
 
-Ubuntu: `sudo apt install openjdk-11-jdk`
+## Teku users 
 
-MacOS: `brew tap AdoptOpenJDK/openjdk && brew cask install adoptopenjdk11`
+See our [user documentation](https://docs.teku.pegasys.tech/en/latest/). 
 
-Other systems: [https://adoptopenjdk.net/] is very helpful. 
+Raise a [documentation issue](https://github.com/PegaSysEng/doc.teku/issues) or get in touch on 
+[Gitter](https://gitter.im/PegaSysEng/teku) if you've got questions or feedback. 
 
-**2) Gradle**
+## Teku developers 
 
-Ubuntu: 
-
-1) Download Gradle
-```shell script
-wget https://services.gradle.org/distributions/gradle-5.0-bin.zip -P /tmp
-sudo unzip -d /opt/gradle /tmp/gradle-*.zip
-```
-2) Setup environment variables
+* [Contribution Guidelines](CONTRIBUTING.md)
+* [Coding Conventions](https://github.com/hyperledger/besu/blob/master/CODING-CONVENTIONS.md)
 
-Create gradle.sh in /etc/profile.d/
-
-```shell script
-export GRADLE_HOME=/opt/gradle/gradle-5.0
-export PATH=${GRADLE_HOME}/bin:${PATH}
-```
-
-Make gradle.sh executable and source the script
+## Build Instructions
 
-```shell script
-sudo chmod +x /etc/profile.d/gradle.sh
-source /etc/profile.d/gradle.sh
-```
+### Install Prerequisites
 
-OSX: `brew install gradle`
+* Java 11
+* Gradle 
```

Comment:
Gradle is not required - the `gradlew` script will automatically retrieve it.

Reply:
Fixed